### DISCUSSION TITLE
로고 사진 화질 문제

### DISCUSSION THREAD

**Discussion by KimGyeongyeon** [boostcampwm-2022/android08-KiwiTalk#17] [General]
스플래시 화면에는 지름 160dp의 원 안에 이미지가 담기는데, 그 사이즈로 줄이면 화질이 망가짐
왼쪽이 원본, 오른쪽이 줄인 것임(윈도우 기본 사진 앱, 인터넷 변환기 모두 비슷했음)

<img width="150" alt="logo_splash_light_288dp" src="https://user-images.githubusercontent.com/67995865/202354048-ecc8fc9d-8482-4bd3-899d-a3709f957039.png">
<img width="150" alt="logo_splash_light" src="https://user-images.githubusercontent.com/67995865/202354050-26aed6f4-6fa3-401a-aef0-9fce50ea3c41.png">

해결 : [공식문서](https://developer.android.com/develop/ui/views/launch/splash-screen#kotlin)에서 사진 규격 관련 안내를 제대로 읽고 해결했음

**Comment by SteelZoo** [comment]
오른쪽은 화면캡쳐 한건가요?

**Reply by KimGyeongyeon** [reply]
Figma에서 export한 왼쪽 사진에서 크기를 세로 160dp로 줄인게 오른쪽입니다

**Comment by choius323** [comment]
![image](https://user-images.githubusercontent.com/60271512/202354970-9e45862b-9a18-4507-afb5-c9c4720eb9d5.png)
이걸로 해보실래요?

**Reply by KimGyeongyeon** [reply]
사이즈가 더 작아야하는데 160으로 줄여주실 수 있나요? 여전히 깨지긴 하는데 전보다는 나은거같아요

**Reply by choius323** [reply]
피그마에서 줄인건데 얼마나 작아야 되는 지 모르겠네요...
가로 세로 160dp로 설정했는데 실제랑 다른 거 같아서 직접 해보시는 게 낫겠네요.

**Reply by KimGyeongyeon** [reply]
오 한버 해볼게요

**Reply by KimGyeongyeon** [reply]
Fail...

**Reply by SteelZoo** [reply]
아니면 안드로이드 스튜디오에서 그냥 이미지 벡터미이지로 변환할수 있었던거 같은데 그걸로 해볼수 있지않을까요?

**Reply by KimGyeongyeon** [reply]
오 문제가 압축이 아니란걸 깨달았어요 성공했습니다
